site/app/libraries/FileUtils.php

```diff
@@ -480,6 +480,43 @@ public static function areWordsInFile(string $file, array $words) {
         return $words_detected;
     }
 
+    public static function checkZipFileStatus($file){
```

Comment:
Let's rename this to `getZipFileStatus`. `check` (to me) implies either the function is going to throw an exception or return true/false.